cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/KafkaSetOperator.java

```diff
@@ -44,6 +44,10 @@ public static boolean needsRollingUpdate(StatefulSetDiff diff) {
             log.debug("Changed volume claim template => needs rolling update");
             return true;
         }
+        if (diff.changesVolumeSize()) {
+            log.debug("Changed size of the volume claim template => no need for rolling update");
+            return false;
```

Comment:
Is the logic right here: The size of the VCT changed, so there's no need for rolling update?

Reply:
This is correct. This is about the template which is part of the pod but has no impact on the existing PVCs. So this would basically do a rolling update of the clusters while not doing anything useful. So I decided it would be best to identify it and ignore it.

Reply:
Thanks for the clarification.